### ISSUE TITLE
Dark Mode Too Far!

### ISSUE THREAD

**Issue by sorens** [adobe-fonts/source-code-pro#267]
Using `Source Code Pro` with Visual Studio Code in Big Sur (just upgraded last night) results in Dark Text.

![image](https://user-images.githubusercontent.com/233764/110658515-9ae8bc80-8176-11eb-8548-3fc5ca607bee.png)

When I switch to another font, the text is readable again

![image](https://user-images.githubusercontent.com/233764/110658307-6e34a500-8176-11eb-9524-a92f3ee5f7ba.png)

# Versions

Visual Studio Code
```
Version: 1.54.1
Commit: f30a9b73e8ffc278e71575118b6bf568f04587c8
Date: 2021-03-04T22:42:18.719Z
Electron: 11.3.0
Chrome: 87.0.4280.141
Node.js: 12.18.3
V8: 8.7.220.31-electron.0
OS: Darwin x64 20.3.0
```

macOS: `11.2.3 (20D91)`

Source Code Pro: https://github.com/adobe-fonts/source-code-pro/releases/tag/2.038R-ro%2F1.058R-it%2F1.018R-VAR

Referenced in issue: https://github.com/microsoft/vscode/issues/100840

**Comment by sorens** [comment]
@frankrolf Interesting! So this morning, I downloaded [2.038](https://github.com/adobe-fonts/source-code-pro/releases/tag/2.038R-ro%2F1.058R-it%2F1.018R-VAR) and installed it. I witnessed no change in behavior (i.e. text was still darkened). In that instance, I just installed the font over the existing one I had. After reading your note, I think maybe that upgrade path was flawed. This time, I removed my original Source Code Pro from Font Book.

![Screen Shot 2021-03-10 at 2 24 42 PM](https://user-images.githubusercontent.com/233764/110706777-3b59d380-81ad-11eb-930d-3b5a0c20629b.png)

Then, I re-downloaded `2.038` and proceeded to install it back into Font Book. Having done that, I ran the same test I used this morning and now the text is visible!

![Screen Shot 2021-03-10 at 2 28 24 PM](https://user-images.githubusercontent.com/233764/110706845-53315780-81ad-11eb-8805-f45c971cb994.png)

Thanks for helping me clarify my issue @frankrolf!

Closing, not a bug.
